### DISCUSSION TITLE
My page won't be created, don't know what i did wrong

### DISCUSSION THREAD

**Discussion by oeduardoramalho** [community/community#60051] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm trying to create a page, but it won't be created even after hours. I thought it was a matter of waiting for it to be built, but it's been almost 8 hours and it still hasn't worked.
I read the documentation to see if I could find some error, but couldn't find anything. Need help.

Here's the repository link: https://github.com/oeduardoramalho/setlists-mma

**Comment by DSDmark** [comment]
Hi @oeduardoramalho,

Did you try these steps: [**How to use page**](https://github.com/orgs/community/discussions/38632#discussioncomment-4092967)

See here, it is working fine when I deploy this: https://dsdmark.github.io/setlists-mma/
So, follow above steps. You haven't done.

**Reply by oeduardoramalho** [reply]
I just tried a little edit, adding my css to a separate folder and it worked. Thanks a lot tho!
